README.md

```diff
@@ -1,9 +1,16 @@
-# SwiftCache
+# swift-resource-provider
 
-A modular cache or data pipeline system.
+A modular resource fetching and management system.
 
-The API is loosely based on Combine/SwiftUI. It doesn't resolve caching logic complexity but helps organize it in a far
-more modular and testable way.
+This little library takes a Combine-like approach to getting things with an identifier. It makes for an easy to
+understand surface for this common abstraction of getting something repeatably based on a series of unique
+characteristics while enabling for progressively adding sophistication to the implementation in composable steps,
+including but not limited to caching steps.
+
+As with many similar frameworks and language facilities, this doesn't make these complicated issues simple but it ought
+to help organize them in a far more modular and testable way.
+
+You can find the full module API documentation at the [Swift Package Index documentation archive](https://swiftpackageindex.com/Gabardone/swift-resource-provider/documentation)
```

Comment:
nice

Reply:
Not there yet but the plan is to move all my packages to swift package index so they get the built and indexed docs without further effort on my part.